pkg/shp/cmd/build/list.go

```diff
@@ -0,0 +1,83 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+
+	"github.com/shipwright-io/cli/pkg/shp/cmd/runner"
+	"github.com/shipwright-io/cli/pkg/shp/params"
+	"github.com/spf13/cobra"
+)
+
+// ListSubCommand struct contains user input to the List subcommand of Build
+type ListSubCommand struct {
+	cmd *cobra.Command
+
+	noHeader *bool
+}
+
+var (
+	writer *tabwriter.Writer
+
+	columnNames    string
+	columnTemplate string
+)
+
+func init() {
```

Comment:
I believe that init() is called when the package is imported, not when the command is run, so this would needlessly run whenever pkg/shp/cmd/build is imported, even if this command is not used.  It should probably just be moved to the func Run() (and the above var() removed as well).

Reply:
You are right, `init` is called when package is imported, I'll fix that, thank you.